src/controllers/auth.controller.js

```
@@ -7,6 +7,7 @@ import mail from "../services/mail.js";
 import User from "../models/User.js";
 import crypto from "crypto";
 import axios from "axios";
+import path from "path";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)